pysmt/operators.py

```diff
@@ -55,7 +55,11 @@
 BV_ASHR,                                    # Arithmetic shift right (47)
 ARRAY_SELECT,                               # Array Select (48)
 ARRAY_STORE,                                # Array Store (49)
-ARRAY_VALUE                                 # Array Value (50)
+ARRAY_VALUE,                                # Array Value (50)
+
+DIV,                                        # Arithmetic Division (51)
```

Comment:
Yes, this is compatible with the tests I made. So, how do you suggest that we proceed? I think that there seems to be a case to leave Div.

Re division by 0: it is undefined (see http://smtlib.cs.uiowa.edu/theories-Reals.shtml)
```Since in SMT-LIB logic all function symbols are interpreted as total
  functions, terms of the form (/ t 0) *are* meaningful in every 
  instance of Reals. However, the declaration imposes no constraints
  on their value. ```